crates/c-api/include/wasmtime/trap.h

```diff
@@ -23,6 +55,16 @@ extern "C" {
  */
 WASM_API_EXTERN wasm_trap_t *wasmtime_trap_new(const char *msg, size_t msg_len);
 
+/**
+ * \brief Attempts to extract the trap code from this trap.
+ *
+ * Returns `true` if the trap is an instruction trap triggered while
+ * executing Wasm. If `true` is returned then the trap code is returned
+ * through the `code` pointer. If `false` is returned then this is not
+ * an instruction trap.
```

Comment:
Could this describe a situation where `false` is returned? (e.g. the trap was originally created with `wasmtime_trap_new`)

Reply:
Yup will do